src/modules/dashboard/components/ActionsPageFeed/ActionsPageEvent/ActionsPageEvent.tsx

```diff
@@ -60,6 +67,7 @@ interface Props {
   emmitedBy?: string;
   actionData: ColonyAction;
   colony: Colony;
+  token?: TokenInfoQuery['tokenInfo'];
```

Comment:
Same here, why do you need an extra prop passed down ?

This is available to you in the `colony` object, but you should know this